### DISCUSSION TITLE
Bladed and OpenFAST

### DISCUSSION THREAD

**Discussion by Unique-Wang** [OpenFAST/openfast#936] [Q&A] [answered]
Dear @jjonkman ,
  
To transform a Bladed model into an OpenFAST model , for example, is it possible to shrink the deviation of blade mass between two models to 100kg?  I used the Bladed data in OpenFAST and the blade mass deviation can reach up to 1000kg, it's a little bit unacceptable. And if possible, where should I start the modification?

Best regards,

**Comment by tatesnow** [comment]
hi @Unique-Wang ,
Do you have the BLADED model of floating wind turbine? Is it possible to share?

**Reply by Unique-Wang** [reply]
没有诶，不好意思哈

**Reply by tatesnow** [reply]
thank you!

**Comment by jjonkman** [comment, marked as answer]
Dear @Unique-Wang,

What structural discretization are you using in your OpenFAST model and Bladed model and does the discrepancy reduce if you increase the structural discretization?  

In OpenFAST, are you using ElastoDyn or BeamDyn to model the blade structural dynamics?

Best regards,

**Reply by Unique-Wang** [reply]
Dear @jjonkman ,

Regarding the first question ,there are 40 blade input stations in my **blade.dat** file, and the discrepancy does reduce if I increase the structural discretization. For the second one ,  I'm using ElastoDyn.

Best regards,

**Reply by Unique-Wang** [reply]
Dear @jjonkman ,

 I successfully shrank the discrepancy down to **0.0x%** by setting **AdjBlMs** in  **balde.dat**  file to the value I want, but I'm not sure if this is the right way.

Best regards,

**Reply by jjonkman** [reply]
Dear @Unique-Wang,

It sounds like the way the distributed structural mass is integrated in Bladed and ElastoDyn to find the total integrated blade mass is a bit different between Bladed and ElastoDyn.  Changing the structural discretization (by adding or removing analysis nodes) and/or setting AdjBlMs are appropriate solutions to ensure that the total integrated mass is the same between the models.

Best regards,
